gallery/src/main.scss

```diff
@@ -15,30 +15,31 @@
 
 .gallery-nav {
   .nx-tree-view__trigger {
+    padding-left: $nx-spacing-xs;
+
     .nx-tree-view__text {
       @include semi-bold();
 
-      color: #393939;
-      font-size: 16px;
+      color: #2c2c2c;
     }
   }
 }
 
 .nx-tree-view__child.gallery-nav-link {
-  height: 32px;
-  line-height: 32px;
+  height: 100%;
+  line-height: $nx-line-height;
   margin-right: 0;
-  margin-bottom: 4px;
+  margin-bottom: 0;
   margin-left: 0;
 }
 
 .gallery-link {
+  @include regular();
+
   border-radius: 6px;
   box-sizing: border-box;
   display: inline-block;
-  height: 32px;
-  margin-bottom: 4px;
-  padding-left: 45px;
+  padding: $nx-spacing-xxs $nx-spacing-xs $nx-spacing-xxs 36px;
```

Comment:
What's the motivation for the non-standard number here?

Reply:
> What's the motivation for the non-standard number here?

With the sidebar spacing, and the indentation of the tree view this was the remainder to get to the indentation number specified in the designs

Reply:
> With the sidebar spacing, and the indentation of the tree view this was the remainder to get to the indentation number specified in the designs

Yes but why was a non-standard number specified in the designs?

Reply:
> > With the sidebar spacing, and the indentation of the tree view this was the remainder to get to the indentation number specified in the designs
> 
> Yes but why was a non-standard number specified in the designs?

From @dturner32's comment in the ticket, "In other words, it should be 52px from the left of the side bar container" 52 is not divisible by 8. As to why maybe @dturner can chime in?